schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -33,11 +33,15 @@
                 },
                 "config": {
                     "$ref": "config.json#/definitions/Config"
+                },
+                "control": {
+                    "$ref": "#/definitions/Control"
                 }
             },
             "required": [
                 "nrfcloud_mqtt_topic_prefix",
-                "pairing"
+                "pairing",
+                "control"
```

Comment:
probably should not be required as it will only be used if the device uses alerts or logging?

Reply:
That is a good point. Should we make it optional for desired as well then?

Reply:
I'm not sure what it being optional for desired means in this case, since it is the cloud side that modifies the desired.control section.

Perhaps 'optional for desired' would mean, if the device does not set reported.control with anything, that would imply it doesn't support alerts or logs.  In that case, the cloud should not include desired.control in deltas for that device.  This is to prevent the situation where when a device does not support the control section, and it gets desired.control via a shadow delta, the device will not correspondingly update reported.control.  That will trigger a second delta whenever the device does update the shadow, since it would not match the desired.control section.  This additional delta would waste data bandwidth and power on the device.

Reply:
Ya that is a good idea. CCing  @sherretten. ☝️